app/models/customer.py

```diff
@@ -0,0 +1,32 @@
+from flask import current_app
+from sqlalchemy.orm import backref
+from app import db
+from sqlalchemy import DateTime
+from datetime import datetime
+from sqlalchemy.orm import relation, relationship
```

Comment:
since we have imported db, we already have access to all of `sqlalchmey` modules. That's why we type everything like `db.Column` or `db.DateTime`

```suggestion
from flask import current_app
from app import db
from datetime import datetime
```